contracts/modules/Exec.sol

```diff
@@ -100,6 +100,31 @@ contract Exec is BaseLogic {
         if (status == DEFERLIQUIDITY__DIRTY) checkLiquidity(account);
     }
 
+    /// @notice Defer liquidity checking for an array of accounts, to perform rebalancing, flash loans, etc. msg.sender must implement IDeferredLiquidityCheck
+    /// @param accounts The array of accounts to defer liquidity for
+    /// @param data Passed through to the onDeferredLiquidityCheck() callback, so contracts don't need to store transient data in storage
+    function deferLiquidityCheckExtended(address[] memory accounts, bytes memory data) external reentrantOK {
```

Comment:
I think better name might be `deferLiquidityChecks` or `deferLiquidityCheckMulti` or something, to  make clear how it was extended.